samanthas_telegram_bot/conversation/auxil/send_to_admin_group.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+from dotenv import load_dotenv
+from telegram import Bot
+from telegram.constants import ParseMode
+
+load_dotenv()
+
+
+async def send_to_admin_group(bot: Bot, text: str, parse_mode: ParseMode | None) -> None:
```

Comment:
You could inline this function. All what it's doing is calling other function

Reply:
I get: "PEP 8: E731 do not assign a lambda expression, use a def" :)

Reply:
You don't need to use lambda :) 
With `inline` I mean replace calling function `send_to_admin_group` (`notify_admins` now) to just call `bot.send_message(admin_group, ...)` since it's function to just call another function.

Pycharm has such feature - you can do it with right mouse click on function, then refactor, then Inline

Reply:
Ah, OK. Done